src/Essentials/src/Email/Email.shared.cs

```diff
@@ -2,9 +2,23 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using System.ComponentModel;
+using Microsoft.Maui.Essentials;
+using Microsoft.Maui.Essentials.Implementations;
 
 namespace Microsoft.Maui.Essentials
 {
+	public interface IEmail
+	{
+		bool IsComposeSupported { get; }
+		
+		Task ComposeAsync();
```

Comment:
What does this task do @Redth ?

Reply:
Blank new email, with no to/subject/etc.